tests/unit_tests/test_margin_calculation.py

```diff
@@ -46,3 +46,86 @@ def test_margin_coinbase_pro():
     assert actual_margin == -11.8
     assert actual_profit == -36.6
     assert actual_sell_fee == 0.96064158
+
+
+def test_calculate_negative_margin_on_binance_when_coin_price_over_1():
```

Comment:
great to add a test.

Reply:
I think they were in @edrickrenan's PR I just preserved them.

Reply:
sure but you did add `test_calculate_negative_margin_on_binance_when_coin_price_over_1` etc.

Reply:
No actually, he removed those from his PR and added new ones Binance and CB.
I did not remove and corrected calculations according to new new precisions.
I added this **test_calculate_negative_margin_on_binance_when_coin_price_over_1_2** also based on @edrickrenan 's question on Telegram.